deploy-prod.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# This script is not currently meant to be executed from within the repository.
+# Copy or symlink it to some deployment directory.
+
+set -ex
+
+cd /var/www/dds
+
+# Update mes-aides
+git fetch origin
+git rebase origin/master
+npm install
+grunt build
+
+# Update openfisca
+npm run install-openfisca
+cat openfisca/api_config.ini | sed s/"port = 2000"/"port = 12000"/ > ~/production.ini
+
+# Restart mes-aides
+sudo restart dds
+
+# Restart openfisca
+sudo start openfisca || sudo restart openfisca
```

Comment:
No idea. There are some differences between the `init.d` services: dds is multi-instance, while openfisca has only one gunicorn process that then manages several openfisca instances.

I don't know if we should really investigate that now. As I said in my first comment, I think this whole mechanism should be replaced by something we understand totally, similar to what we use for staging. The intended scope of this PR was just to solve some concrete issues that may cause bugs in deployment (like we had yesterday for 10min), and we will definitely need later more work on the production.